service/authorization/authorization.go

```diff
@@ -175,7 +175,28 @@ func (as *AuthorizationService) GetDecisions(ctx context.Context, req *authoriza
 			// get attribute definition/value combinations
 			dataAttrDefsAndVals, err := retrieveAttributeDefinitions(ctx, ra, as.sdk)
 			if err != nil {
-				// TODO: should all decisions in a request fail if one FQN lookup fails?
+				// if attribute an FQN does not exist
+				// return deny for all entity chains aginst this RA set and continue to next
+				if strings.Contains(err.Error(), db.ErrTextNotFound) {
```

Comment:
Can we somehow use `errors.Is` here instead of seeking? I suspect this is about as performant, so I'm okay if this is the recommended solution

Reply:
I don't think those db errors actually implement the error interface. We should probably fix that if we plan to leverage these in other places besides policy. 

cc: @jakedoublev

Reply:
The [`db.ErrTextNotFound` is an error message](https://github.com/opentdf/platform/blob/1d76303af08bc1f479f7163a32141a5506ebe6ea/service/pkg/db/errors.go#L101) passed to [`status.Error()` here along with the not found code](https://github.com/opentdf/platform/blob/1d76303af08bc1f479f7163a32141a5506ebe6ea/service/pkg/db/errors.go#L116-L118), and that function signature [expects a code and a message](https://github.com/grpc/grpc-go/blob/v1.64.0/status/status.go#L58).

It's the safe error message, not a Go error. Perhaps a rename to `db.ErrMsgTextNotFound` as the safe external RPC error message would be the right call? And possibly checking for the code of the response as `codes.NotFound` instead of the error message string?